### PR TITLE
Remove enableComponentStackLocations

### PR DESCRIPTION
**Pull request by rickhanlonii**
This has landed everywhere

### CODE REVIEW

File: packages/shared/ReactComponentStackFrame.js [hunk 1]
```diff
@@ -296,28 +290,12 @@ export function describeNativeComponentFrame(
...
 export function describeClassComponentFrame(ctor: Function): string {
-  if (enableComponentStackLocations) {
-    return describeNativeComponentFrame(ctor, true);
-  } else {
-    return describeFunctionComponentFrame(ctor);
-  }
+  return describeNativeComponentFrame(ctor, true);
 }
 
 export function describeFunctionComponentFrame(fn: Function): string {
```
Comment (rickhanlonii) [inline packages/shared/ReactComponentStackFrame.js:297]: @sebmarkbage we need to keep this old export to support old devtools versions right?

**Review by sebmarkbage — APPROVED** [review]
Lol. We're even on the next version after this one now with `enableOwnerStacks`.

File: packages/shared/ReactComponentStackFrame.js (hunk 1 above)
Comment (sebmarkbage) [reply packages/shared/ReactComponentStackFrame.js:297]: DevTools has its own copy in `DevToolsComponentStackFrame` so it can be deleted.

Comment (sebmarkbage) [reply packages/shared/ReactComponentStackFrame.js:297]: This export seems used by other places still though.

Comment (rickhanlonii) [reply packages/shared/ReactComponentStackFrame.js:297]: I missed this before merging somehow: https://github.com/facebook/react/pull/31781

**Pull request merged**